src/renderer/controllers/torrent-list-controller.js

```diff
@@ -220,7 +220,8 @@ module.exports = class TorrentListController {
 
     menu.append(new electron.remote.MenuItem({
       label: 'Save Torrent File As...',
-      click: () => dispatch('saveTorrentFileAs', torrentSummary.torrentKey)
+      click: () => dispatch('saveTorrentFileAs', torrentSummary.torrentKey),
+      enabled: typeof torrentSummary.torrentFileName !== 'undefined'
```

Comment:
This can just be `torrentSummary.torrentFileName !== undefined`. Or, even better, treat `null` the same way too: `torrentSummary.torrentFileName != null`.

`typeof x !== 'undefined'` is only needed if the variable might not exist at all (to prevent a TypeError)